### PR TITLE
Update Zabbbix documentation with more accurate instructions to use the webhook

### PR DESCRIPTION
**Pull request by sysy6r**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

This impact the zabbix documentation page : https://docs.datadoghq.com/fr/integrations/zabbix/

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: zabbix/README.md [hunk 1]
```diff
@@ -34,7 +34,19 @@ For Agent v7.21+ / v6.21+, follow the instructions below to install the Zabbix c
 ##### Create Datadog media type 
 
 1. Navigate to *Administration > Media Types > Create Media Type*.
-2. Add Datadog api_key as a parameter. And, add the following Zabbix template variables as parameters: {ALERT.MESSAGE}, {ALERT.SUBJECT}, {EVENT.DATE}, {EVENT.NAME}, {EVENT.NSEVERITY}, {EVENT.TAGSJSON}, {EVENT.TIME}, {EVENT.VALUE}, {ITEM.NAME}
+2. Add paramters to the webhook using Zabbix template variables : Add your Datadog api_key and, and the following Zabbix template variables as parameters:
```
Comment (maycmlee) [inline zabbix/README.md:37]:
```suggestion
2. Add parameters to the webhook using Zabbix template variables. Add your Datadog api_key and the following Zabbix template variables as parameters:
```

File: zabbix/README.md [hunk 2]
```diff
@@ -34,7 +34,19 @@ For Agent v7.21+ / v6.21+, follow the instructions below to install the Zabbix c
...
+        event_date |  {EVENT.DATE}
+        event_name | {EVENT.NAME}
+        event_nseverity |  {EVENT.NSEVERITY}
+        event_tags | {EVENT.TAGSJSON}
+        event_time | {EVENT.TIME}
+        event_value | {EVENT.VALUE}
+        item_name | {ITEM.NAME}
+        alert_message | {ALERT.MESSAGE}
+        alert_subject | {ALERT.SUBJECT}
+   ```
```
Comment (maycmlee) [inline zabbix/README.md:49]: Can we put this info into a table instead?

File: zabbix/README.md [hunk 3]
```diff
@@ -34,7 +34,19 @@ For Agent v7.21+ / v6.21+, follow the instructions below to install the Zabbix c
 ##### Create Datadog media type 
 
 1. Navigate to *Administration > Media Types > Create Media Type*.
-2. Add Datadog api_key as a parameter. And, add the following Zabbix template variables as parameters: {ALERT.MESSAGE}, {ALERT.SUBJECT}, {EVENT.DATE}, {EVENT.NAME}, {EVENT.NSEVERITY}, {EVENT.TAGSJSON}, {EVENT.TIME}, {EVENT.VALUE}, {ITEM.NAME}
+2. Add paramters to the webhook using Zabbix template variables : Add your Datadog api_key and, and the following Zabbix template variables as parameters:
+   ```
+        api_key | "Enter Yours"
```
Comment (maycmlee) [inline zabbix/README.md:39]:
```suggestion
        api_key | Your Datadog API key
```

**Review by maycmlee — CHANGES_REQUESTED** [review]

**Comment by fanny-jiang** [comment]
Thanks for opening this PR! Looks like the `conf.yaml.example` is out of sync since this [change](https://github.com/DataDog/integrations-extras/pull/1147) was merged. Can you please run `ddev -e validate config zabbix -s` to sync the config file?

**Pull request merged**